server/src/main/java/org/opensearch/wlm/WlmMode.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm;
+
+import org.opensearch.common.annotation.PublicApi;
+
+/**
+ * Enum to hold the values whether wlm is enabled or not
+ */
+@PublicApi(since = "2.18.0")
+public enum WlmMode {
+    ENABLED("enabled"),
+    MONITOR_ONLY("monitor_only"),
```

Comment:
This is same as `SearchBackpressureMode`. What does it mean for one to be enabled and other disabled? How can we seamlessly transition users from `SearchBackpressure` to `WorkloadManagement`?

Reply:
They both can be enabled but in that case SearchBackpressure will only work for default query group tasks.

Reply:
We can make that change in the same PR to account for it.

Reply:
Should be consider SearchBackpressure disabled, if WLM is enabled? I am assuming WLM can also act on DEFAULT_QUERY_GROUP tasks?

Reply:
I think SBP should be enabled initially to kill potential run away tasks with no query group or invalid one.